osi_lane.proto

```diff
@@ -1034,6 +1034,18 @@ message LaneBoundary
             // A structure (e.g. building or tunnel wall).
             //
             TYPE_STRUCTURE = 13;
+
+            // A barrier (e.g. a concrete barrier on a highway).
+            //
+            TYPE_BARRIER = 14;
+
+            // A sound barrier.
+            //
+            TYPE_SOUND_BARRIER = 15;
+
+            // A railing (e.g. divider between road and sidewalk).
```

Comment:
Question: Is this sufficiently distinguished from guard_rail? I.e. a guard rail can also be considered a divider between road and sidewalk in certain situations, however should be mapped to TYPE_GUARD_RAIL, so maybe this should be more clearly stated?